scripts/make_release.ps1

```diff
@@ -36,6 +36,7 @@ RunGoBuild "github.com/pulumi/pulumi/sdk/v2/go/pulumi-language-go" "sdk" "pulumi
 CopyPackage "$Root\sdk\nodejs\bin" "pulumi"
 
 Copy-Item "$Root\sdk\nodejs\dist\pulumi-resource-pulumi-nodejs.cmd" "$PublishDir\bin"
+Copy-Item "$Root\sdk\nodejs\dist\pulumi-python-shim.cmd" "$PublishDir\bin"
```

Comment:
Note that we need the pulumi-python-shim.cmd packaged as part of the plugin and be installed in PATH. Based on what I saw, this should be sufficient but I am not familiar enough with the mechanics of the build/publish for windows to know for sure this works. My crappy windows VM can't handle the build for pulumi so I am leaving this to CI to build and will validate the published zip to check if the script makes it in. FYI @justinvp